tooling/nargo_cli/src/cli/verify_cmd.rs

```diff
@@ -48,10 +51,19 @@ pub(crate) fn run(
         Some(NOIR_ARTIFACT_VERSION_STRING.to_string()),
     )?;
 
+    let mut workspace_file_manager = file_manager_with_stdlib(&workspace.root_dir);
+
     let expression_width = backend.get_backend_info()?;
     for package in &workspace {
-        let program =
-            compile_bin_package(&workspace, package, &args.compile_options, expression_width)?;
+        // TODO: Same as other todo, we can be adding files for the entire workspace
```

Comment:
Not clear on this, seems like it's an old TODO which was asking for the same change as in `fmt_cmd.rs`?

Reply:
yep -- resolved